src/common/cloud-control/Cargo.toml

```diff
@@ -0,0 +1,28 @@
+[package]
+name = "common-cloud-control"
```

Comment:
Why put `common-cloud` to `common`?

Reply:
It's a crate of `query`?

Reply:
I think it is not purely depend on databend-query as it is an individual RPC client, thus we could put it under common directory more better modularity

Reply:
That's ok